docs/contributing/contributing.qmd

```diff
@@ -7,7 +7,7 @@ please be aware of our [code of conduct](code_of_conduct.qmd).
 
 # Prerequisites
 
-To start, clone DASCore and [install it in development mode](dev_install.qmd).
+To start, follow the [recipes](https://dascore.org/recipes/overview.html) for the contribution and clone DASCore and [install it in development mode](dev_install.qmd).
```

Comment:
So there is quite a bit of redundant information from what is in the recipe vs already on the contributor docs, but I think the recipe is really useful for someone who is new to git, and contributing code in general. So perhaps we phrase it like this:

```
To start, clone DASCore and [install it in development mode](dev_install.qmd). If you are new to contributing to open-source projects, this [recipe](recipes/how_to_contribute.qmd) provides step by step instructions. 
```

Also note how you can link directly to the qmd file rather than the rendered html.